src/org/usfirst/frc/team694/robot/subsystems/Drivetrain.java

```diff
@@ -218,3 +233,34 @@ public boolean isCalibrating() {
     }
 
 }
+=======
+    
+    public void resetAccelerometer() {
+        accelerometer.reset();
+    }
+    
+    public double getXAccel() {
+        return accelerometer.getWorldLinearAccelX();
+    }
+    
+    public double getYAccel() {
+        return accelerometer.getWorldLinearAccelY();
+    }
+    
+    public double getZAccel() {
+        return accelerometer.getWorldLinearAccelZ();
+    }
+    
+    public double getZRotation() {
+        return accelerometer.getYaw();
+    }
+    
+    public boolean testForBump() {
+        return getZAccel() > -1;
```

Comment:
While you're at it, add a constant inside Drivetrain that holds a threshold for when we're crossing a bump